genesis.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "gossamer",
+  "id": "gossamer",
+  "bootNodes": [
+    "/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj"
+  ],
+  "protocolId": "gossamer",
```

Comment:
Also any idea what protocolId is? I know we're mimicing the other file, but I'm just curious

Reply:
it's the protocol used by the p2p layer